addons/nuxeo-template-rendering/nuxeo-template-rendering-core/src/main/resources/OSGI-INF/listener-contrib.xml

```diff
@@ -16,7 +16,7 @@
       <event>aboutToRemove</event>
     </listener>
 
-    <listener name="documenttemplate-type-binding" async="true" priority="500"
+    <listener name="documenttemplate-type-binding" async="true" postCommit="true" priority="500"
```

Comment:
But here the listener is asynchronous, I find it confusing to add postCommit=true.
Shouldn't we use postCommit="true" only when async=false and the listener extends PostCommitEventListener, which means nowhere since the goal of the ticket is to remove them?

Reply:
As the post commit behavior is driven by the fact of extending `PostCommitEventListener`, no matter the attribute, see https://github.com/nuxeo/nuxeo/blob/master/nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/event/impl/EventListenerDescriptor.java#L150.

So yes, the attribute is useless, and we could remove it everywhere.

Yet, while the deprecation of the PostCommitEventExecutor mechanism is not done, I find it better to make it clearly appear in the XML contribution (and there are still test contributions with `postCommit="true" async="false"`).

That's why I suggest to:
- Set postCommit="true" explicitly to `true` where unset or badly set to `false` with a class extending `PostCommitEventListener`.
- Set postCommit="false" where set to true for listeners not implementing PostCommitEventListener.

Reply:
okay